source/game/system/CourseMap.cc

```diff
@@ -51,6 +54,17 @@ MapdataCannonPointAccessor *CourseMap::parseCannonPoint(u32 sectionName) {
     return accessor;
 }
 
+MapdataGeoObjAccessor *CourseMap::parseGeoObj(u32 sectionName) {
+    const MapSectionHeader *sectionPtr = m_course->findSection(sectionName);
+
+    MapdataGeoObjAccessor *accessor = nullptr;
+    if (sectionPtr) {
```

Comment:
Perhaps we could shorten to:
```suggestion
    return sectionPtr ? new MapdataGeoObjAccessor(sectionPtr) : nullptr;
```

I know this doesn't match the other functions though...

Reply:
#134 opened for this. I do not think this should be handled by this PR.